src/card/CardImage.tsx

```diff
@@ -10,7 +10,7 @@ const CardImage: RneFunctionComponent<ImageProps> = ({ style, ...props }) => {
 
 const styles = StyleSheet.create({
   image: {
-    width: null,
+    width: undefined,
```

Comment:
it must be null

Reply:
okay, then finding a better sloution for this.

Reply:
Muzhe bhi batana😅